### PR TITLE
Updated the generated files

### PR DESCRIPTION
**Pull request by jaym**

### CODE REVIEW

**Comment by yzl** [comment]
lgtm

File: lib/omnibus/generator_files/Gemfile.erb [hunk 1]
```diff
@@ -13,9 +13,10 @@ gem 'omnibus', '~> <%= Omnibus::VERSION.split('.')[0...-1].join('.') %>'
 # by running `bundle install --without development` to speed up build times.
 group :development do
```
Comment (schisamo) [inline lib/omnibus/generator_files/Gemfile.erb]: I wonder if we should do away with this `:development` group and recommend people use ChefDK?

Comment (jaym) [inline lib/omnibus/generator_files/Gemfile.erb]: i'm in the bundle all the things camp.

File: lib/omnibus/generator_files/Gemfile.erb [hunk 2]
```diff
@@ -13,9 +13,10 @@ gem 'omnibus', '~> <%= Omnibus::VERSION.split('.')[0...-1].join('.') %>'
...
   # Use Berkshelf for resolving cookbook dependencies
-  gem 'berkshelf', '~> 3.0'
+  gem 'berkshelf', '~> 3.3'
 
   # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'test-kitchen',    '~> 1.2'
-  gem 'kitchen-vagrant', '~> 0.14'
+  gem 'test-kitchen',    '~> 1.4'
+  gem 'kitchen-vagrant', '~> 0.18'
+
```
Comment (sethvargo) [inline lib/omnibus/generator_files/Gemfile.erb]: newline

**Comment by schisamo** [comment]
@jaym Be sure to give the `README.md.erb` and `omnibus.rb.erb` files a look over and ensure there aren't any new config options we should include. The `windows_arch` option jumps to mind.

**Comment by jaym** [comment]
@schisamo i added `windows_arch`
I left it uncommented so we do not display deprecation warnings

**Comment by schisamo** [comment]
@jaym We should also ensure the boxes in the [`.kitchen.yml.erb`](https://github.com/chef/omnibus/blob/b98d79189e939148edff599d42aa627a9f4a3f0e/lib/omnibus/generator_files/.kitchen.yml.erb#L15-L38) match up with the current bento releases:
https://github.com/chef/bento#current-baseboxes

**Comment by schisamo** [comment]
This looks ready for merge.

**Pull request merged**